cds_ils/config.py

```diff
@@ -654,3 +654,5 @@ def _parse_env_bool(var_name, default=None):
 CDS_ILS_INDEX_LOCAL_ACCOUNTS = True
 
 CDS_ILS_MIGRATION_ALLOW_UPDATES = False
+
+CDS_ILS_FALLBACK_CREATION_DATE = date(2010, 1, 1).isoformat()
```

Comment:
please do not use use config when it is not to meant to be configured.
Create a file `consts.py` or similar and just hardcoded, then change all occurrences to use such value and not config.